source/MonoGame.Extended/KNI.Extended.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>It makes KNI more awesome.</Description>
+    <PackageTags>kni extended pipeline bmfont tiled texture atlas input viewport fps shapes sprite</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Remove="MonoGame.Framework.DesktopGL" />
```

Comment:
Should we move the MG PackageReference inside .csproj files?

Reply:
Instead of adding the package references inside of every Kni.Extended.* csproj you can instead update the `Directory.Build.props` to the following

```xml
<ItemGroup Condition="!$(DefineConstants.Contains('KNI'))">
    <PackageReference Include="MonoGame.Framework.DesktopGL"
                        Version="3.8.1.303"
                        PrivateAssets="All" />
</ItemGroup>

<ItemGroup Condition="$(DefineConstants.Contains('KNI'))">
    <PackageReference Include="nkast.Xna.Framework" Version="3.12.9001" />
    <PackageReference Include="nkast.Xna.Framework.Content" Version="3.12.9001" />
    <PackageReference Include="nkast.Xna.Framework.Graphics" Version="3.12.9001" />
    <PackageReference Include="nkast.Xna.Framework.Input" Version="3.12.9001" />
    <PackageReference Include="nkast.Xna.Framework.Game" Version="3.12.9001" />
</ItemGroup>
```

Then in each of the Kni.Extended.* csproj files you can add the following

```xml
<PropertyGroup>
    <DefineConstants>$(DefineConstants);KNI</DefineConstants>
</PropertyGroup>
```

This will filter it correctly and we only have to keep track of the nkast package reference in one place (the props file) instead of every csproj when they need updating in the future.